modules/programs/sftpman.nix

```
@@ -0,0 +1,101 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.sftpman;
+
+  jsonFormat = pkgs.formats.json { };
+
+  mountOpts = { config, name, ... }: {
+    options = {
+      host = mkOption {
+        type = types.str;
+        description = "The host to connect to.";
+        default = null;
```

I generally think it is best to place the fields in the order `type`, `default`, `apply`, `example`, `description`:

```suggestion
        type = types.nullOr types.str;
        default = null;
        description = "The host to connect to.";
```

Note, if you specify `types.str` then `null` is not an allowed value. You need to use `nullOr str`.